src/components/Navbar/SearchDrawer/PreInput/SearchQuerySuggestion/SearchQuerySuggestion.module.scss

```diff
@@ -12,3 +14,10 @@
     background-color: var(--color-background-alternative-deep);
   }
 }
+
+.removeIcon {
```

Comment:
we're using `.removeIcon` class instead of `<Button>` component because the Button component doesn't support height and width `spacing-small` ? is that the case?

Reply:
Yes when using Button even for small size it takes too much space also the hovering effect is off